### PR TITLE
SAK-43089 remove counter to display the time alert when using prev/next

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4986591&bid=16464702)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4986591&bid=16464702)

File: samigo/samigo-app/src/webapp/js/saveForm.js [hunk 1]
```diff
@@ -179,7 +174,7 @@ function SaveFormContentAsync(toUrl, formId, buttonName, updateVar, updateVar2,
         // when the request is done the scope of the function can be garbage collected...
     }
 
-    if (doautosave && counter > 1) {
+    if (doautosave) {
```
Comment (adrianfish) [inline samigo/samigo-app/src/webapp/js/saveForm.js]: doautosave is always true, it seems. It gets set to true and nothing else touches it.

Comment (adrianfish) [reply samigo/samigo-app/src/webapp/js/saveForm.js]: Can I add a commit onto this, just doing some cleanup? Variables that are only accessed once, formatting all over the place. I can shorten this file and make it a lot clearer.

Comment (ottenhoff) [reply samigo/samigo-app/src/webapp/js/saveForm.js]: I thought it was overridden from a JSP .... but I can't find any overrides. So sure, let's remove it. But I'd rather not dramatically re-format this JS .... it's super sensitive and this code is super prone to regressions

Comment (jonespm) [reply samigo/samigo-app/src/webapp/js/saveForm.js]: Yeah, looking at this it doesn't look like it's used, it looks like it has always been true. It's not passed in from deliverAssessment (SaveFormContentAsync) and if this is called it's going to be true. This is really hard to read code for sure.

Comment (jonespm) [reply samigo/samigo-app/src/webapp/js/saveForm.js]: I somewhat expected Codacy to catch this. It doesn't but it also really doesn't like this file. https://app.codacy.com/manual/sakaiproject/sakai/file/42082279127/issues/source?bid=16464702&fileBranchId=16464702

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4986591&bid=16464702)

**Pull request merged**